### DISCUSSION TITLE
Unable to install Windows into iSCSI target due to some UEFI bugs?

### DISCUSSION THREAD

**Discussion by imyzcn** [ipxe/ipxe#600] [Q&A] [answered]
Hi, folks,

Recently I came across this issue exactly the same as this one ‘https://github.com/ipxe/ipxe/discussions/305'.

The sanhook directive in UEFI mode kept on reporting an error saying ‘Could not describe SAN devices: Operation not supported (http://ipxe.org/3c222083)’, while it works fine in Legacy/BIOS mode. I understand that the parameter ‘—no-describe’ may remove that error, and the side-effect to the iBFT is obvious too.

Is there any workaround to address this problem?? FYI, here is my software environment:

* VMware Workstation 16.2.2
* Tiny PXE Server 1.0.0.23 (DHCP, Wimboot, Option17, ipxe scripts & etc)
* ipxe-x86_64.efi: iPXE 1.21.1+ (g16d95)
* VM WinServer 2022: iSCSI Target host
* VM Client: UEFI, ethernet0.virtualDev = "e1000e"
* Network: All of those VMs are within a same LAN segment.

**Comment by mcb30** [comment, marked as answer]
This issue is unrelated to #305.  It looks as though the VMware UEFI BIOS has a bug, that would need to be fixed by VMware.  The notes at https://ipxe.org/err/3c2220 accurately describe the problem and the limitations of the available workarounds.

**Reply by imyzcn** [reply]
Thank you for your prompt reply. It sounds as if nothing can be done but turn to VMware. Anyway, one less thing to worry about, which is a good news to me indeed. Have a nice day!
